scenarios/hitech95/mail-generic-bf.yaml

```diff
@@ -0,0 +1,30 @@
+# Global brouteforce
+type: leaky
+#debug: true
+name: crowdsecurity/email-generic-bf
+description: "Detect generic email brute force"
+filter: "evt.Meta.log_type == 'mail_auth' && evt.Meta.sub_type == 'auth_fail'"
+groupby: evt.Meta.source_ip
+capacity: 5
+leakspeed: "10s"
+blackhole: 1m
+labels:
+ service: email
+ type: bf
+ remediation: true
+---
+# Per user brouteforce
+type: leaky
+#debug: true
+name: crowdsecurity/email-user-bf
```

Comment:
Please change `crowdsecurity` to `hitech95`

Reply:
Should be done 😃